web/src/main/resources/application.properties

```diff
@@ -97,3 +97,5 @@ csp.frameancestor.allow.scheme=https
 intygsadmin.url=http://localhost:8081
 intygsadmin.cache.expiry=86400
 intygsadmin.cron=0 * * * * ?
+
+inactivate.previous.major.version.for.certificate.type=ts-bas,ts-diabetes
```

Comment:
Should we really have inactive certificate types set in default properties? 

I would prefer that we leave the default value empty and that it is only set in configmap for the specific environment. Sure, it adds additional configuration but it is more visible what is configured for a certain environment instead of "hidden" in default properties.

Reply:
Moved configuration fromd default to dev.

Reply:
This should also be updated. minaintyg\devops\openshift\test\configmap-envvar.properties

Reply:
minaintyg\devops\openshift\test\configmap-envvar.properties is just for dintyg. No updated needed.